includes/Order/Hooks.php

```diff
@@ -263,10 +263,19 @@ public function on_sub_order_change( $order_id, $old_status, $new_status, $order
         // return if any child order is not completed
         $all_complete = true;
 
+        // Exclude manual gateways from auto complete order status update for digital products.
+        $excluded_gateways = apply_filters( 'dokan_order_status_update_exclusion_payment_gateways', array( 'bacs', 'cheque', 'cod' ) );
```

Comment:
Plse rename `dokan_order_status_update_exclusion_payment_gateways` filter name to `dokan_excluded_payment_gateways_on_order_status_update`